docs/book/features/orphan_snapshots.md

```diff
@@ -0,0 +1,71 @@
+## Orphan snapshots detection and clean-up
+From vSphere CSI plugin's perspective, orphan snapshots are vSphere snapshots that were initiated through the vSphere CSI driver but do not have a corresponding VolumeSnapshotContent object in Kubernetes clusters on the vCenter.
```

Comment:
Can you change "vSphere CSI plugin" to "vSphere CSI driver" to be consistent?

Reply:
done